src/protections/RoomTakedown/DiscoveredRoomStore.ts

```diff
@@ -76,19 +73,9 @@ export class StandardDiscoveredRoomStore
       if (details.creator === undefined) {
         continue; // no point persisting details that don't have a creator.
       }
-      const storeResult = await this.hashStore.storeRoomIdentification({
-        creator: details.creator,
-        roomID: details.room_id,
-        server: roomIDServerName(details.room_id),
-      });
-      if (isError(storeResult)) {
-        log.error(
-          "Error storing room details for a room",
-          roomID,
-          details,
-          storeResult
-        );
-      }
+      // TODO: normally we would store details about the room, such as the creator,
+      // and the server, but currently this is unviable.
+      // https://matrix.org/blog/2025/07/security-predisclosure/
```

Comment:
Ok this grand idea doesn't work, we actually do depend on being able to extract the creator in tests.. even though in reality this code is probably populating the database with bad data....

Reply:
You could always provide a bogus server name if it's not critical

Reply:
Ok so all of these changes were unecessary lemao